### PR TITLE
Port PR https://github.com/wso2/carbon-apimgt/pull/6466

### PR DESCRIPTION
**Pull request by ruwiniwj**
Port PR  https://github.com/wso2/carbon-apimgt/pull/6466 (issue wso2/product-apim#4960)

This also includes PR #6385(issue wso2/product-apim#4865)

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/services/logout/logout_callback.jag [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<%
```
Comment (pubudu538) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/services/logout/logout_callback.jag]: Let's add the license header

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Base/index.jsx [hunk 2]
```diff
@@ -34,60 +42,60 @@ import VerticalDivider from '../Shared/VerticalDivider';
 
 const styles = theme => ({
     // New styles ..........
-    ///////////////////////////////////
-    ///////////////////////////////////
+    // /////////////////////////////////
+    // /////////////////////////////////
```
Comment (pubudu538) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Base/index.jsx]: Let's remove these commented code.

Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Base/index.jsx]: Fixed

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/services/logout/logout_callback.jag (hunk 1 above)
Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/services/logout/logout_callback.jag]: Fixed

**Pull request merged**